thoth_user_api/api_v1.py

```diff
@@ -193,8 +193,31 @@ def post_buildlog(log_info: dict):
     }, 202
 
 
+def list_runtime_environments(page: int=0):
+    """List available runtime environments"""
+    graph = GraphDatabase()
+    graph.connect()
+
+    result = graph.runtime_environment_listing(page, page+PAGINATION_SIZE)
+    return {
+        "results": result,
+        "results_count": len(result),
+        "page": page,
+        "page_size": PAGINATION_SIZE
+    }, 200
+
+
+def get_runtime_environment(runtime_environment_name: str, analysis_id: str=None):
+    """Get packages inside the given runtime environment."""
+    graph = GraphDatabase()
+    graph.connect()
+
+    result = graph.get_runtime_environment(runtime_environment_name, analysis_id)
+    return list(map(lambda x: x.to_pretty_dict(), result)), 200
```

Comment:
Raise 404 if not found - this has to be raised as not found exception on adapter side.